agentMET4FOF/agents.py

```diff
@@ -1611,7 +1614,8 @@ class MonitorAgent(AgentMET4FOF):
         Used to specifically select only a few keys to be plotted
     """
 
-    def init_parameters(self, plot_filter=[], custom_plot_function=-1, **kwargs):
+    def init_parameters(self, plot_filter=[], custom_plot_function=-1, *args, **kwargs):
+        # super(MonitorAgent, self).init_parameters()
```

Comment:
this is a good intention. however, for the *args to work upon instantiation of agents, we will need to add it to the _add_osbrain_agent and _add_mesa_agent functions and pass the *args appropriately. Inside these functions, init_parameters are also called.

Reply:
if you could remove the commented super method, that'd be great as well to reduce clutter

Reply:
I've removed the commented code. Should the *args be removed for now as well?